lib/pdk/cli/util/changelog_generator.rb

```diff
@@ -0,0 +1,58 @@
+module PDK
+  module CLI
+    module Util
+      class ChangelogGenerator
+        def generate_changelog
+          retries = 0
+          result = nil
+          loop do
+            result = system('bundle exec rake changelog')
+            retries += 1
+            break if result || retries > 3
```

Comment:
This doesn't make any sense.  Running the same command three times makes it magically work?

Reply:
I guess there is no need for a retry here, therefore I will remove it. Initially had networking issues with this but the errors were not very explicit. Thanks.